Address comment from ISEG review (#34)
boucadair/add-resolver-information: draft-ietf-add-resolver-info.md

      error codes that can be returned by this DNS resolver. A value can be an individual EDE or a range of EDEs. Range values MUST be identified by "-".  When
      multiple non-contiguous values are present, these values MUST be comma-separated.

      Returned EDEs (e.g., Blocked (15), Censored (16), and Filtered (17)) indicate whether the DNS resolver is configured to reveal the reason why a query was filtered/blocked, when such event happens. If the resolver's capabilities are updated to include new similar
      error codes, the resolver can terminate the TLS session, prompting the client to initiate a new TLS connection and retrieve the
      resolver information again. This allows the client to become aware of the resolver's updated capabilities. Alternatively, if the

>       client receives an EDE for a DNS request that does not match the error codes in exterr, the client can query the resolver again to

Reviewer: ```suggestion
      client receives an EDE for a DNS request, but that EDE was not listed in  the "exterr", the client can query the resolver again to
```

Reviewer: If we maintain this text (and TLS session), I think we need to delete "Once a resolver is selected by a DNS client, this document does not interfere with DNS operations with that resolver." from the introduction.

>       learn about the updated resolver's capabilities to return new error codes. If a mis-match still exists, the client can identify that

Reviewer: ```suggestion
      learn about the updated resolver's capabilities to return new error codes. If a mismatch still exists, the client can identify that
```
